README.md

```diff
@@ -97,36 +100,34 @@ match result {
         let endpoint: SocketAddr = packet.addr();
         let received_data: &[u8] = packet.payload();
 
-        // you can here deserialize your bytes into the data you have passed it when sending.
+        // You can deserialize your bytes here into the data you have passed it when sending.
 
         println!("Received packet from: {:?} with length {}", endpoint, received_data.len());
     }
     Ok(None) => {
-        println!("This could happen when we have'n received all data from this packet yet");
+        println!("This could happen when we have not received all the data from this packet yet");
     }
     Err(e) => {
-        // We get an error if something when wrong like the address is already in use.
+        // We get an error if something went wrong, like the address is already in use.
         println!("Something went wrong when receiving, error: {:?}", e);
     }
 }
-```
-## Notice
-"This library is not yet stable. It is experimental and things may change frequently."
-
-### Contribution
 
-Unless you explicitly state otherwise, any contribution intentionally submitted
-for inclusion in the work by you, as defined in the Apache-2.0 license, shall be dual licensed as above, without any
-additional terms or conditions.
+```
 
 ## Authors
 
 - [Lucio Franco](https://github.com/LucioFranco)
 - [Fletcher Haynes](https://github.com/fhaynes)
 - [Timon Post](https://github.com/TimonPost)
 
-We want to especially give credits to [gaffer on games](https://gafferongames.com/). 
-This library is based on his vision about how an gamenetwork protocol should work.
+We want to give credit to [gaffer on games](https://gafferongames.com/) as we have used his guide to building a game networking protocol to build this library. 
```

Comment:
Do we give credit to "gaffer on games" or "gaffer". Maybe better to give credits to his name than a bunch of articles.

Reply:
It does, but I think we can keep it cuz its the name of the website.